src/pybind/mgr/dashboard/frontend/src/app/shared/components/form-advanced-fieldset/form-advanced-fieldset.component.ts

```diff
@@ -1,10 +1,12 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'cd-form-advanced-fieldset',
   templateUrl: './form-advanced-fieldset.component.html',
   styleUrls: ['./form-advanced-fieldset.component.scss']
 })
 export class FormAdvancedFieldsetComponent {
+  @Input()
+  customTitle: string = 'Advanced';
```

Comment:
simply `title` is enough
```suggestion
  title: string = 'Advanced';
```

Reply:
i think we do use it in couple of places in the dashboard already so i don't see a problem with it.
```
╰─$ git grep "title: string"                                                                       128 ↵
src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-details/iscsi-target-details.component.ts:  title: string;
src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.ts:  title: string;
src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.ts:  title: string;
src/pybind/mgr/dashboard/frontend/src/app/shared/components/vertical-navigation/vertical-navigation.component.ts:  @Input() title: string;
src/pybind/mgr/dashboard/frontend/src/app/shared/forms/crud-form/crud-form.model.ts:  title: string;
src/pybind/mgr/dashboard/frontend/src/app/shared/models/chart-tooltip.ts:      titleLines.forEach((title: string) => {
src/pybind/mgr/dashboard/frontend/src/app/shared/models/chart-tooltip.ts:  getTitle(title: string) {
src/pybind/mgr/dashboard/frontend/src/app/shared/models/osd-deployment-options.ts:  title: string;
src/pybind/mgr/dashboard/frontend/src/app/shared/services/notification.service.ts:  
```

Reply:
but if you see any problem while using, we can reconsider it

Reply:
Updated to title